### ISSUE TITLE
[Bug]: Query for Individual Ratings Returning Users Without Reviews

### ISSUE THREAD

**Issue by coglebed** [nsunami/postreview-app#83]
### URL of the Page

https://postreview-app.herokuapp.com/articles/1d38c6a6-124f-4ff6-a496-22d7bbd930f7

### What happened?

The icons above the subraitings are currently missing 
![image](https://user-images.githubusercontent.com/73071333/151013467-4e3edceb-829e-49ac-9518-ca009f024fa2.png)

### Steps to reproduce

1. submit a review
2. get to the paper page

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by nsunami** [comment]
The problem here is that the database query is returning users without reviews submitted for the article. Will investigate.

**Comment by nsunami** [comment]
I believe this was fixed by #91 
@coglebed Can you try to reproduce this issue?

**Comment by coglebed** [comment]
@nsunami 
Tried to replicate the issue, but was unable to create the necessary state for it.
I needed to create a rating card with some sub-rating scores missing to test how it would render, but when I submit the fully empty rating - the page just refreshes as if nothing happened (scenario 1) or if I try to submit a rating with >0 but less than a complete set of ratings - it just goes into loading screen forever (scenario 2). In either case - the incomplete rating cannot be generated, so in a way, this is no longer a possible issue (although we might need to one day figure out a real solution for that if we want incomplete ratings to be submitted).
